app/components/Form/ProjectQuarterlyReportFormSummary.tsx

```diff
@@ -0,0 +1,160 @@
+  if (!renderDiff)
+    quarterlyReportFormChanges =
+      summaryProjectQuarterlyReportFormChanges.edges.filter(
+        ({ node }) => node.operation !== "ARCHIVE"
+      );
+
+  // Sorting the quarterly report form changes by the reporting requirement index
+  const [sortedQuarterlyReports] = useMemo(() => {
+    const filteredReports = quarterlyReportFormChanges.map(({ node }) => node);
+
+    filteredReports.sort(
+      (a, b) =>
+        a.newFormData.reportingRequirementIndex -
+        b.newFormData.reportingRequirementIndex
+    );
+
+    return [filteredReports];
+  }, [quarterlyReportFormChanges]);
+
+  // Defines if all quarterly reports are pristine
+  const allFormChangesPristine = useMemo(
+    () =>
+      !quarterlyReportFormChanges.some(
+        ({ node }) => node?.isPristine === false || node?.isPristine === null
+      ),
+    [quarterlyReportFormChanges]
+  );
+
+  const quarterlyReportsJSX = useMemo(() => {
+    return sortedQuarterlyReports.map((quarterlyReport, index) => {
+      if (!quarterlyReport) return;
+
+      // Set custom rjsf fields to display diffs
+      const customFields = { ...fields, ...CUSTOM_DIFF_FIELDS };
+
+      // Set the formSchema and formData based on showing the diff or not
+      const { formSchema, formData } = !renderDiff
+        ? {
+            formSchema: projectReportingRequirementSchema,
+            formData: quarterlyReport.newFormData,
+          }
+        : getFilteredSchema(
+            projectReportingRequirementSchema as JSONSchema7,
+            quarterlyReport
+          );
+
+      return (
+        <div key={index} className="reportContainer">
+          <header>
+            <h4>Quarterly Report {index + 1}</h4>
+          </header>
+          {/* Show this part if non of quarterly report form properties have been updated */}
+          {Object.keys(formSchema.properties).length === 0 &&
+            quarterlyReport.operation !== "ARCHIVE" && (
+              <em>Quarterly report not updated</em>
+            )}
+          {/* Show this part if the whole quarterly report has been removed */}
+          {renderDiff && quarterlyReport.operation === "ARCHIVE" ? (
+            <em className="diffOld">Quarterly Report Removed</em>
+          ) : (
+            <FormBase
+              liveValidate
+              key={`form-${quarterlyReport.id}`}
+              tagName={"dl"}
+              theme={readOnlyTheme}
+              fields={renderDiff ? customFields : fields}
+              schema={formSchema as JSONSchema7}
+              uiSchema={quarterlyReportUiSchema}
+              formData={formData}
+              formContext={{
+                operation: quarterlyReport.operation,
+                oldData:
+                  quarterlyReport.formChangeByPreviousFormChangeId?.newFormData,
+              }}
+            />
+          )}
+          <style jsx>{`
+            .reportContainer {
+              margin-bottom: 1em;
+            }
+            diffOld {
```

Comment:
`.diffOld` (missing .)

Reply:
@mikevespi haha, that style was totally useless; I've removed that and nothing changed 😄